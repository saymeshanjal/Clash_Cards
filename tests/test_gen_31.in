99
14 388 4
10 73 3
2 236 1
10 56 5
8 156 5
10 43 5
6 248 4
14 101 4
6 237 2
16 202 1
14 288 1
8 269 4
8 333 2
6 297 3
10 76 1
8 64 1
16 122 2
2 329 3
4 3 5
10 50 2
16 208 3
14 267 2
14 271 5
8 49 4
16 382 5
2 81 2
8 184 1
16 248 5
4 386 3
16 254 5
2 182 1
4 252 3
16 395 1
6 94 1
14 205 3
14 341 1
16 241 5
8 8 1
16 145 2
16 137 3
2 137 2
6 34 2
2 1 2
10 67 1
6 33 3
14 23 5
8 7 1
4 106 1
16 201 2
16 344 4
12 89 3
6 43 2
8 27 3
16 360 2
4 146 5
2 313 3
12 396 2
8 181 5
6 316 2
6 387 3
8 105 4
12 99 3
4 174 5
14 325 2
14 198 4
16 398 4
16 116 5
16 247 2
2 49 3
8 371 3
16 246 5
8 163 4
8 108 5
16 130 2
6 200 5
2 351 3
16 255 1
8 222 1
10 6 2
4 114 4
6 41 5
10 324 4
4 112 4
10 231 5
10 193 2
6 194 1
16 97 1
16 204 3
16 392 1
4 82 5
14 108 1
4 129 3
2 146 2
10 383 3
6 315 1
2 94 2
14 165 2
8 196 1
14 340 1
