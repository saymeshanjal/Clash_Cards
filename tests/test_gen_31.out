910 1520 1002
110 142 0
1 1 0
45 207 0
8 62 0
45 207 0
1 14 5
910 1520 1002
5 6 9
6435 6435 0
1716 1716 0
8 42 20
20 21 29
6 14 0
126 126 0
35 35 0
4081 4082 4707
0 2 0
0 6 0
76 77 99
6307 6563 0
1078 1079 1275
987 2445 0
8 42 20
4248 8622 0
0 1 1
35 35 0
4248 8622 0
1 5 0
4248 8622 0
1 1 0
1 5 0
6435 6435 0
10 10 0
1652 1780 0
1716 1716 0
4248 8622 0
31 39 0
4081 4082 4707
6307 6563 0
0 1 1
5 6 9
0 1 1
126 126 0
6 14 0
36 3396 0
33 37 0
3 3 0
4081 4082 4707
3809 5406 3655
430 494 0
5 6 9
27 43 0
4081 4082 4707
0 6 0
0 2 0
286 287 351
8 62 0
5 6 9
6 14 0
8 42 20
430 494 0
0 6 0
1078 1079 1275
910 1520 1002
3809 5406 3655
4248 8622 0
4081 4082 4707
0 2 0
27 43 0
4248 8622 0
8 42 20
8 62 0
4081 4082 4707
1 19 0
0 2 0
6435 6435 0
35 35 0
1 81 170
0 5 1
1 19 0
44 133 75
0 5 1
45 207 0
76 77 99
10 10 0
6435 6435 0
6307 6563 0
6435 6435 0
0 6 0
1716 1716 0
1 5 0
0 1 1
110 142 0
10 10 0
0 1 1
1078 1079 1275
35 35 0
1716 1716 0
